.travis.yml

```diff
@@ -30,7 +30,7 @@ env:
         - MAIN_CMD='python setup.py'
         - SETUP_CMD='test'
         - EVENT_TYPE='pull_request push'
-        - PIP_DEPENDENCIES=''
+        - PIP_DEPENDENCIES='gwcs'
```

Comment:
good to know. I'll dig up the logs as see what went wrong, there were quite a few issues recently.